test/page.spec.js

```diff
@@ -1653,6 +1654,15 @@ module.exports.addTests = function({testRunner, expect, headless}) {
       });
       expect(screenshot).toBeGolden('screenshot-grid-fullpage.png');
     });
+    it('should take fullPage screenshots when defaultViewport is null', async({server}) => {
+      const browser = await puppeteer.launch({ args: ['--no-sandbox'], defaultViewport: null });
+      const page = await browser.newPage();
+      await page.goto(server.PREFIX + '/grid.html');
+      const screenshot = await page.screenshot({
+        fullPage: true
+      });
+      expect(screenshot).toBeGolden('screenshot-grid-fullpage-2.png');
```

Comment:
Size of screenshot is different when `defaultViewport` is null - it's why I've created additional fixture. Is that okay ?

Reply:
Yes, perfect.